src/main/java/car/model/CarName.java

```diff
@@ -0,0 +1,59 @@
+package car.model;
+
+import car.view.Constant;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class CarName {
+    private List<String> carNames;
+
+    public void setCarName(String inputCarNames){
+        validateNull(inputCarNames);
+        List<String> carNames = Arrays.asList(inputCarNames.split(","));
+        valid(carNames);
+        this.carNames = carNames;
+    }
+
+    public List<String> getCarNames() {
+        return carNames;
+    }
+
+    private void valid(List<String> carNames) {
```

Comment:
검증 역할을 하는 다른 메서드들은 `validate`라는 워딩을 사용하는데요. 일관성있게 수정하면 어떨까요!
```suggestion
    private void validate(List<String> carNames) {
```